package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
```

Comment:
Looks like you have a good start to your file structure. Remember to add "  
```
"overrides": {
    "autoprefixer": "10.4.5"
  },
```
"
 to your package.json file to avoid issues with deploying on Netlify.